src/components/com_tjcertificate/media/js/tjCertificate.js

```diff
@@ -0,0 +1,67 @@
+jQuery(document).ready(function() {

```

Comment:
Can you write JS using Objects like other JS file in the folder

Reply:
done